File: anycompany/src/test/java/za/co/anycompany/model/Customer.java
package za.co.anycompany.model;

import java.util.Date;

public class Customer {
    private String name;
    private String country;
    private Date dateOfBirth;
    private int Id;

    public int getId() {
        return Id;
    }
    public void setId(int Id) {
    	this.Id = Id;
    }
    
    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getCountry() {
        return country;
    }

    public void setCountry(String country) {
        this.country = country;
    }

    public Date getDateOfBirth() {
        return dateOfBirth;
    }

    public void setDateOfBirth(Date dateOfBirth) {
        this.dateOfBirth = dateOfBirth;
    }
}

File: AnyCompanyEntertainment/AnyCompanyEntertainment/model/Customer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AnyCompanyEntertainment_.model
{
    public class Customer
    {
        private String name;
        private String country;
        private DateTime dateOfBirth;

        public String getName()
        {
            return name;
        }

        public void setName(String name)
        {
            this.name = name;
        }

        public String getCountry()
        {
            return country;
        }

        public void setCountry(String country)
        {
            this.country = country;
        }

        public DateTime getDateOfBirth()
        {
            return dateOfBirth;
        }

        public void setDateOfBirth(DateTime dateOfBirth)
        {
            this.dateOfBirth = dateOfBirth;
        }
    }

}
